README.md

```diff
@@ -263,63 +269,46 @@ This section provides a step-by-step procedure for deploying ODIMRA and GRF plug
+   odimra   30326 30304  0 Jan07 ?        00:41:25 ./svc-account-session --registry=consul --registry_address=consul:8500 --server_address=odimra:45101
+   root     30344 30077  0 Jan07 ?        00:00:00 sudo -E -u odimra nohup ./svc-aggregation --registry=consul --registry_address=consul:8500 --server_address=odimra:45102 --client_request_timeout=300s
+   root     30366 30077  0 Jan07 ?        00:00:00 sudo -E -u odimra nohup ./svc-api --registry=consul --registry_address=consul:8500 --client_request_timeout=302s
+   odimra   30374 30301  0 Jan07 ?        00:01:42 ./svc-update --registry=consul --registry_address=consul:8500 --server_address=odimra:45108 --client_request_timeout=300s
+   odimra   30375 30291  0 Jan07 ?        00:31:24 ./svc-task --registry=consul --registry_address=consul:8500 --server_address=odimra:45105 --client_request_timeout=300s
+   odimra   30381 30344  0 Jan07 ?        00:04:27 ./svc-aggregation --registry=consul --registry_address=consul:8500 --server_address=odimra:45102 --client_request_timeout=300s
+   odimra   30398 30306  0 Jan07 ?        00:01:25 ./svc-fabrics --registry=consul --registry_address=consul:8500 --server_address=odimra:45106 --client_request_timeout=300s
+   odimra   30399 30314  0 Jan07 ?        00:03:13 ./svc-systems --registry=consul --registry_address=consul:8500 --server_address=odimra:45104 --client_request_timeout=300s
+   odimra   30414 30321  0 Jan07 ?        00:01:14 ./svc-managers --registry=consul --registry_address=consul:8500 --server_address=odimra:45107 --client_request_timeout=300s
+   odimra   30426 30366  0 Jan07 ?        00:17:52 ./svc-api --registry=consul --registry_address=consul:8500 --client_request_timeout=302s
    ```
 
 
-    **NOTE:**
-    - The resource aggregator configuration files are available at `/etc/odimra_config`.
-    -  The GRF configuration files are available at `/etc/grf_plugin_config`.
-    - The resource aggregator API service runs on the default port 45000.
-    - The GRF plugin API service runs on default port 45001.
-    - The resource aggregator logs are available at `/var/log/odimra`.
-    - The GRF plugin logs are available at `/var/log/GRF_PLUGIN`.
+  **NOTE:**
+  - The resource aggregator configuration files are available at `/etc/odimra_config`.
+  - The GRF configuration files are available at `/etc/grf_plugin_config`.
+  - The URP configuration files are available at `/etc/urp_plugin_config`.
+  - The resource aggregator API service runs on the default port 45000.
+  - The GRF plugin API service runs on the default port 45001.
+  - The URP API service runs on the default port 45003.
+  - The resource aggregator logs are available at `/var/log/odimra`.
+  - The GRF plugin logs are available at `/var/log/GRF_PLUGIN`.
+  - The URP logs are available at `/var/log/URP_PLUGIN`.
```

Comment:
URP logs are not at this location..we can check logs with docker command 'docker logs build_urp_1'

Reply:
Updated this line.